### PR TITLE
Corrections due to Markdown spell check errors

### PR DESCRIPTION
**Pull request by StephanStS**
@fpetru: I did some corrections (e.g. no single trailing spaces at the end of a line) and some indentation issues.

Shall I also do the corrections to dietpi_tools.md?  
Expand "Check Markdown syntax' there: <https://github.com/MichaIng/DietPi-Docs/runs/1271785423>

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -9,23 +9,23 @@
...
 
-:    Companion application to Sonarr and Radarr, which manages and downloads subtitles based on your requirements, now available for install. Open [Bazarr](../software/bittorrent#bazarr) page in [Optimised software](../software/).
+ Companion application to Sonarr and Radarr, which manages and downloads subtitles based on your requirements, now available for install. Open [Bazarr](../software/bittorrent#bazarr) page in [Optimised software](../software/).
     Many thanks to @DiogoAbu for doing this suggestion [Issue #2045](https://github.com/MichaIng/DietPi/issues/2045)
 
-- **Docker logging available in RAM/journald** 
-    This feature is now available to newly fresh Docker installs or reinstalls. Logs are now done with limited verbosity to systemd-journald (RAM). They could be accessible running next command: 
-    
-``` bash
+- **Docker logging available in RAM/journald**
```
Comment (MichaIng) [inline docs/release-notes.md:15]: Here are two trailing white spaces 😛.

File: docs/release-notes.md [hunk 2]
```diff
@@ -9,23 +9,23 @@
     ![DietPi-Software Bazarr](assets/images/dietpi-software_bazarr.jpg)
     For more details on installation and configuration open [DietPi Optimised Software - Bazarr](../software/bittorrent#bazarr) page.
 
-:    Companion application to Sonarr and Radarr, which manages and downloads subtitles based on your requirements, now available for install. Open [Bazarr](../software/bittorrent#bazarr) page in [Optimised software](../software/).
+ Companion application to Sonarr and Radarr, which manages and downloads subtitles based on your requirements, now available for install. Open [Bazarr](../software/bittorrent#bazarr) page in [Optimised software](../software/).
```
Comment (MichaIng) [inline docs/release-notes.md:12]: There are now sometimes ones, sometimes two and sometimes three white spaces to keep paragraphs as part of the list elements. I suggest to use two in every case, which is the default single tab indentation and that way the text starts exactly below the text of behind the bullet point:
```
- abc
  def
```

**Comment by StephanStS** [comment]
> Ah I thought this was already done since no syntax error here: https://github.com/MichaIng/DietPi-Docs/runs/1269114295
> Since the page renders fine, nothing urgent but yes would be awesome if we could get everything fixed or intentionally accepted+excluded it there is a good reason.

@MichaIng: This #94 was my last commit which led to a error free run. Commit #95 added new points I would then correct (markdown issues, link issues and spelling issues).  
See <https://github.com/MichaIng/DietPi-Docs/runs/1271687986?check_suite_focus=true> which was the run after #95.

File: docs/release-notes.md (hunk 2 above)
Comment (StephanStS) [reply docs/release-notes.md:12]: The number of the leading white spaces I have chosen depending on what shall be displayed.  
I use the atom editor with its internal preview (via ctrl-shift-m, the first ctrl-shift-m always fails...). I made the experience that atom has a very good Markdown preview (not mkdocs preview!) compared e.g. to haroopad etc.

I did the changes using the atom markdown preview function and also propose to edit the releasenotes in this way.

File: docs/release-notes.md (hunk 1 above)
Comment (fpetru) [reply docs/release-notes.md:15]: @StephanStS @MichaIng yes - if possible please do the changes. many thanks

**Review by fpetru — APPROVED** [review]

**Pull request merged**